security/tls-authorization/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>infinispan-simple-tutorials-security-tls-authorization</artifactId>
+    <parent>
+        <relativePath>../../pom.xml</relativePath>
+        <version>1.0.0-SNAPSHOT</version>
+        <groupId>org.infinispan.tutorial.simple</groupId>
+        <artifactId>infinispan-simple-tutorials</artifactId>
+    </parent>
+    <name>Infinispan Simple Tutorials: Remote</name>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <executable>java</executable>
+                    <arguments>
+                        <argument>-Djava.net.preferIPv4Stack=true</argument>
+                        <argument>-classpath</argument>
+                        <classpath />
+                        <argument>org.infinispan.tutorial.simple.security.tls.InfinispanRemote</argument>
+                    </arguments>
+                </configuration>
```

Comment:
you can run the scripts here
```suggestion
                 <executions>
                    <execution>
                        <id>generate-certificates</id>
                        <phase>generate-resources</phase>
                        <goals>
                            <goal>exec</goal>
                        </goals>
                        <configuration>
                            <executable>${basedir}/create_unsigned_certificates.sh</executable>
                            <executable>${basedir}/create_signed_certificates.sh</executable>
                            <executable>${basedir}/create_signed_server_truststore_auth.sh</executable>
                        </configuration>
                    </execution>
                    <execution>
                        <id>run</id>
                        <goals>
                            <goal>exec</goal>
                        </goals>
                        <configuration>
                            <executable>java</executable>
                            <arguments>
                                <argument>-Djava.net.preferIPv4Stack=true</argument>
                                <argument>-classpath</argument>
                                <classpath />
                                <argument>org.infinispan.tutorial.simple.security.tls.InfinispanRemote</argument>
                            </arguments>
                        </configuration>
                    </execution>
                </executions>
```

Reply:
@wfink Should we use Pedro's suggestion here?